package.json

```diff
@@ -58,6 +58,7 @@
     "rollup-plugin-esbuild": "^6.1.1",
     "rollup-plugin-filesize": "^10.0.0",
     "simple-git-hooks": "^2.11.1",
+    "tsx": "^4.16.2",
```

Comment:
Why?

Reply:
just for testing purpose

Reply:
It should be removed it before the merge